packages/client/src/runtime/core/types/GetResult.ts

```diff
@@ -43,8 +43,15 @@ export type GetFindResult<P extends Payload, A> =
   | { include: infer S } & Record<string, unknown>
   ? {
       [K in keyof S as S[K] extends false | undefined | null ? never : K]:
-        S[K] extends true
+        S[K] extends object
```

Comment:
Flips the condition around, instead of checking if it's a boolean, we check if it's an object. This then has the side-effect of treating `true` and `boolean` the same. More details on what and why [here](https://prisma-company.slack.com/archives/C016KUHB1R6/p1685727223766319?thread_ts=1685699415.404239&cid=C016KUHB1R6).